### ISSUE TITLE
[Feature Request] Options for always open in new tab

### ISSUE THREAD

**Issue by emisjerry** [zk4/obsidian-evergreen#3]
could it be possible that there's a option for toggle "Always open in new tab"? for every hyperlinks?

**Comment by songxianpeng** [comment]
+1 from me

**Comment by zk4** [comment]
> could it be possible that there's a option for toggle "Always open in new tab"? for every hyperlinks?

I think that's the default behavior without evergreen plugin .

**Comment by emisjerry** [comment]
No, you have to press Ctrl+Click, then the note opened in new tab.
Without Ctrl, it will open in the current tab.

**Comment by zk4** [comment]
Maybe you can just use evergreen plugin for 'always open in new tab'.  If I make a toggle for this feature. It would break the workflow of evergreen note. Definitely not what I want.
